tests/neg/namedTypeParams.scala

```diff
@@ -17,11 +17,13 @@ object Test {
   def f[X, Y](x: X, y: Y): Int = ???
 
   f[X = Int, String](1, "") // error // error
-  f[X = Int][X = Int][Y = String](1, "") // error: illegal repeated type application
+  /* Conflicts with Clause Interweaving, stems from named type parameters assuming one type clause
```

Comment:
Did you have a look at what it would entail to fix named type arguments?

Reply:
After an offline discussion, I realized that named type arguments are also an experimental feature, and that in most cases you could use interleaved clauses _instead_ of named type arguments, but you are unlikely to use both at the same time.

Reply:
Since there is a lot of overlap between clause interleaving and named type arguments, and since the latter has apparently been rejected, and should not become part of the language.
Therefore the compatibility between them will not be looked at.